vars/signArtifacts.groovy

```diff
@@ -0,0 +1,25 @@
+void call(Map args = [:]) {
+
```

Comment:
[For the end to end scenario, not this PR]

In order to call this function from the promotion script it needs to be in the form of a closure - that part can come afterward when the sha512 changes are made.

It would could make sense to keep this file as is built and create another function that returns the closure version.